context-free.md

```diff
@@ -19,3 +23,4 @@ order to turn those rules into a picture?_
 
 ## What do you think is interesting about the ContextFree program you wrote?
 
+I believe the ContextFree program that I wrote was fascinating because it only has 3 rules, 2 of which function recursively and each one works in terms of either itself or another. In spite of being only a few short lines of code thus defined in terms of nothing more than black and white primitive shapes, it is able to generate an exceedingly intricate fractal with several branches with possibly several thousands of shapes created in perfect symmetry by using recursion and a loop simultaneously. I also believed it to be interesting since to me, this - using this DSL to generate patterns next to impossible to design by hand or a photo-editor - is its purpose and thus in creating a complex image, I was able to use this DSL the way I believed it to be intended, therefore making it in my opinion an incredible learning experience for my first ever usage of a DSL.
```

Comment:
Compared to your graph, mine sucks. And it uses even fewer lines than mine. But that is the fascinating part of this language I guess: tell the computer the rule and let it do the repeat job. One fascinating thing is I do not need to import anything and the final code looks so simple, compared to JAVA.